code/controllers/subsystems/zcopy.dm

```diff
@@ -316,14 +316,14 @@ SUBSYSTEM_DEF(zcopy)
 		if (istype(A, /atom/movable/openspace/overlay))
 			var/atom/movable/openspace/overlay/OO = A
 			var/atom/movable/AA = OO.associated_atom
-			out += "<li>\icon[A] plane [A.plane], layer [A.layer], depth [OO.depth], associated Z-level [AA.z] - [OO.type] copying [AA] ([AA.type], eventually [OO.mimiced_type])</li>"
+			out += "<li>[CHAT_ICON(A)] plane [A.plane], layer [A.layer], depth [OO.depth], associated Z-level [AA.z] - [OO.type] copying [AA] ([AA.type], eventually [OO.mimiced_type])</li>"
```

Comment:
These aren't going to chat.

Reply:
I can find and replace it to HTML_ICON if that would make you happy. I don't think it really matters.

Reply:
That would be fine; main thing is that its usage matches its name.